### PR TITLE
v1.0.0 release

### PR DESCRIPTION
**Pull request by YegorZaremba**
@navendu-pottekkat Please test this PR

Changes:
- [x] Improve lazy loading logic #21 
- [x] Improve handling of dynamically rendering DOM elements #24 
- [x] Add ESLint and setup CI #27

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -15,62 +15,75 @@
...
+  for (let i = 0; i < images.length; i++) {
+    clasifyImage(images[i])
+  }
 }
 
 // Call function when DOM is loaded
-window.addEventListener("load", () => {
-  clasifyImages()
+document.addEventListener('DOMContentLoaded', () => {
+  // @docs some websites only works with manual recheck
```
Comment (navendu-pottekkat) [inline src/content.js:62]: What do you mean by manual recheks?

File: src/background.js [hunk 2]
```diff
@@ -19,60 +19,60 @@ import 'regenerator-runtime/runtime'
...
-          If the top predicition is in our filter list, filter the image (return true)
-          */
-          console.log(op[0].className, op[0].probability)
-          return true;
-        }
-        else {
-          return false;
+        const result = op[0] && op[0].className && FILTER_LIST.includes(op[0].className)
+
+        // lazy loading handler, if falase main src url, do race for first nsfw true result
```
Comment (navendu-pottekkat) [inline src/background.js:65]: Could you make it more clear?

File: src/content.js (hunk 1 above)
Comment (navendu-pottekkat) [reply src/content.js:62]: Could you make the comment more descriptive?

**Comment by navendu-pottekkat** [comment]
@YegorZaremba Please check my comments

**Comment by YegorZaremba** [comment]
> I have tested it out. Looks good apart from the small lag. But I do not find that off-putting because people using this extension would not normally search for NSFW content directly as we do in our tests. There is no lag in the browser.

We block all images, and only after that starts analyze, check with google image for any safe search result - you see the lag too
And yeah, that an issue which should be fixed asap

I also added handler for case where backround worker is not alive yet, so looks like we are ready to merge

**Comment by navendu-pottekkat** [comment]
@YegorZaremba Sure, I will just check it out and we can merge

**Comment by navendu-pottekkat** [comment]
@YegorZaremba Could you check this and merge the PR?

**Comment by YegorZaremba** [comment]
> Looks good, I think we can merge, what about npm-run script watch, can we use it. It should be changed in the docs. Also, I think we can set the version in manifest.json to 1.0.0, add you to the authors' list.
I upped version, add myself as contributor and remove `watch` script, dunno why we need it

> Also, should we need to build all the files in src/?
Sure, cause I `content.js` and `background.js` use common code in `utils.js` so we should build all code, otherwise it not gonna work

**Comment by navendu-pottekkat** [comment]
Okay! You can merge it. Looks like we are all done. I will make changes to docs in the master branch itself. You can merge and we can talk about the release!

**Comment by YegorZaremba** [comment]
Hope we don't have heavy memory leaks, my brief tests show we are fine

**Comment by YegorZaremba** [comment]
v1.0.0 Merged

**Pull request merged**